squad/frontend/templates/squad/index.jinja2

```diff
@@ -1,34 +1,58 @@
 {% extends "squad/base.jinja2" %}
 
 {% block content %}
-<h2 class="page-header well">{{ _('All groups') }}</h2>
 
 {% if user.is_authenticated %}
 <div style='margin-top: 2em; margin-bottom: 2em;'>
     <a class='btn btn-default' href="{{url("new-group")}}">{{ _('New group') }}</a>
 </div>
-
 {% endif %}
-<div class='highlight-row'>
-{% for group in groups %}
-<div class="row row-bordered project">
-    <a href="{{group_url(group)}}">
-    <div class="col-md-3 col-sm-3">
-        <strong>
-            {{group.display_name}}
-        </strong>
+
+
+{% set all_groups = {_('All groups'): all_groups, _('User spaces'): user_spaces} %}
+{% for title, groups in all_groups.items()  %}
+    <h2 class="page-header well">{{ title  }}</h2>
+    <div>
+        <form action="/" method="GET">
+            <label class="form-check-label" for="order">{{ _('Order by') }}:</label>
+            <input
+              name="order"
+              value="most_active"
+              type="radio"
+              onchange="$(this.form).submit()"
+              {{ 'checked' if request.GET.get('order') != 'by_name' else ''}}
+            />
+            <span>{{ _('most active') }}</span>&nbsp;
```

Comment:
"last updated"?

Reply:
Ops, you mentioned that in the comment above, didn't catch it :)

Updated it now